hawkular-inventory-parent/hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/model/Metric.java

```diff
@@ -104,26 +110,35 @@ public static Builder builder() {
     @JsonInclude(Include.NON_NULL)
     private final MetricUnit unit;
 
-    @DocModelProperty(description = "Metric labels.",
+    @DocModelProperty(description = "Metric expression used to evaluate the metric value. If not set, assume $metric.",
```

Comment:
I think this should always be set, right?

Reply:
if we want to make it required, we can.